### PR TITLE
Show if maker has xmr autoconf enabled

### PR DESCRIPTION
**Pull request by chimp1984**
Implements https://github.com/bisq-network/bisq/issues/4569

### CODE REVIEW

File: core/src/main/resources/i18n/displayStrings.properties [hunk 1]
```diff
@@ -338,7 +338,7 @@ offerbook.offerersAcceptedBankSeats=Accepted seat of bank countries (taker):\n {
 offerbook.availableOffers=Available offers
 offerbook.filterByCurrency=Filter by currency
 offerbook.filterByPaymentMethod=Filter by payment method
-offerbook.timeSinceSigning=Time since signing
+offerbook.timeSinceSigning=Signed since
```
Comment (ripcurlx) [inline core/src/main/resources/i18n/displayStrings.properties:341]: Did you change this to save space?

**Comment by ripcurlx** [comment]
@chimp1984 How about using the fast clock instead of the rocket?
Bildschirmfoto 2020-09-30 um 17 58 33

**Comment by chimp1984** [comment]
Yes ok as well, though i felt the rocket might add more incentive is looks a bit more "special"... but feel free to use the clock, no strong opinion here...

**Comment by chimp1984** [comment]
Maybe the clock could be used for all instant trade options?

**Comment by ghost** [comment]
Expressing my preference for the rocket!  I think it looks cool and may generate interest in the feature...

image

**Comment by chimp1984** [comment]
Yes I think it sticks out a bit more than a more traditional icon. The "to the moon" crew will like it as well ;-)

**Comment by ripcurlx** [comment]
> Yes I think it sticks out a bit more than a more traditional icon. The "to the moon" crew will like it as well ;-)

ok - voted down and I accept my defeat 😜

**Review by ripcurlx — APPROVED** [review]
ACK

As described in https://github.com/bisq-network/bisq/issues/4569 this sets for xmr trades the autoconf flag on offer creation. It won't be updated if the config is changed after the offer has been created.

**Pull request merged**